app/__init__.py

```diff
@@ -1,11 +1,43 @@
 import os
 from flask import Flask, render_template, request
 from dotenv import load_dotenv
+import folium
+import json
 
 load_dotenv()
 app = Flask(__name__)
 
 
 @app.route('/')
 def index():
```

Comment:
My main concern here would be that the route serving method can get too complex if we add more and more things in the same method. So it might be better to separate this implementation into smaller methods, and use it from this route serving method.

e.g. Following is an example method that can be separated, and then you can have another method like `create_map`, and all of these can be called from this route method. It should make the code a bit more cleaner imo

```
def load_member_locations():
    locations_file_path = Path(__file__).resolve().parent / 'static' / 'data' / 'locations.json'
    with open(locations_file_path, 'r') as f:
        data = json.load(f)
        members = data['members']
    return members
```